workflow/controller/taskset.go

```diff
@@ -136,9 +136,8 @@ func (woc *wfOperationCtx) reconcileTaskSet(ctx context.Context) error {
 		for nodeID, taskResult := range workflowTaskSet.Status.Nodes {
 			node, err := woc.wf.Status.Nodes.Get(nodeID)
 			if err != nil {
-				woc.log.Warnf("[SPECIAL][DEBUG] returning but assumed validity before")
-				woc.log.Errorf("[DEBUG] Was unable to obtain node for %s", nodeID)
-				return err
+				woc.log.Info("Continuing to retry")
```

Comment:
we should definitely be more specific here -- if you read this message in the Controller's logs, it would be pretty ambiguous.

```suggestion
				woc.log.Debugf("Unable to obtain node for %s during TaskSet Reconciliation. Expected during a retry, continuing", nodeId)
```

I also think `debug` level makes more sense here, since this is now an _expected_ error